### ISSUE TITLE
Enhancement of the Password Generator project

### ISSUE THREAD

**Issue by payallenka** [Mrinank-Bhowmick/python-beginner-projects#334]
A strong password should have punctuation mark like '@' or '#' in the program even tough punctuations are being called but in the generated password they are not used.

I'd like to add this feature that every generated password will be a combination of alphabets (lower and caps), numbers and special characters.

**Comment by Ayushlion8** [comment]
Hey @payallenka, I would like to work on this issue please assign it to me:)

**Comment by payallenka** [comment]
Yes @Mrinank-Bhowmick , I would like to collaborate with @Ayushlion8. Kindly assign this issue to both of us.

**Comment by th-shristi** [comment]
@Mrinank-Bhowmick I have made a PR on this kindly look into it and label it under Hacktoberfest
